kitchen/bin/kitchen

```diff
@@ -913,6 +913,7 @@ if __name__ == '__main__':
 
     parser = argparse.ArgumentParser(description='A toy HTTP Service for testing the Waiter platform')
     parser.add_argument('--hostname', metavar='HOSTNAME', default='', help='Server host name')
+    parser.add_argument('--log-output', metavar='LOG_OUTPUT', choices=['stdout', 'stderr', 'file'], default='stdout', help='Log output destination')
```

Comment:
Should we make `file` the default for backwards compatibility?

Reply:
None of our tests are using the `kitchen.log` file for anything. Our Clojure version of Kitchen wrote to the console ([source](https://github.com/twosigma/waiter/blob/c3575bd/kitchen/resources/log4j.properties)), so we can consider this as fixing backward compatibility with the old Kitchen code. I believe I made the change originally because we weren't originally capturing *stdout* and *stderr* into files on Kubernetes. Now we emulate the Mesos behavior by creating those files with our `waiter-init` script. I think changing the output to `kitchen.log` was a bad change in hindsight.